### ISSUE TITLE
Export doesn't return the specified API anymore

### ISSUE THREAD

**Issue by FlorianKlosPpi** [jedmao/linez#21]
The export of the linez package does not return the API anymore, but an object with the API as the `default` property.

_Should:_
```
{ [Function: linez]
  Document: [Function: Document],
  configure: [Function: configure],
  resetConfiguration: [Function: resetConfiguration] }
```
_Is:_
```
{ default: 
   { [Function: linez]
     Document: [Function: Document],
     configure: [Function: configure],
     resetConfiguration: [Function: resetConfiguration] } }
```

This breaks for example the usage in the eclint package:
```
Unhandled rejection TypeError: linez is not a function
  at /xxx/node_modules/eclint/js/eclint.js:114:27
```

**Comment by jedmao** [comment]
Can you show me your import statement?

**Comment by jedmao** [comment]
Just published the fix in v4.1.4. Thanks for reporting!
